core/network/impl/protocols/protocol_fetch_chunk.hpp

```diff
@@ -45,9 +45,9 @@ namespace kagome::network {
                                     host,
                                     make_protocols(kFetchChunkProtocol,
                                                    genesis_hash,
-                                                   "polkadot"),
+                                                   chain_spec),
```

Comment:
[rust](https://github.com/paritytech/polkadot/blob/3fd990501423edca102bc473c2bb1c2ce9b93c23/node/network/protocol/src/request_response/mod.rs#L277-L286)
```suggestion
                                                  "polkadot"),
```